parsers/test/test_EIA.py

```diff
@@ -0,0 +1,189 @@
+import os
```

Comment:
Awesome that you added tests for this. 

On contrib, the guidelines for mocking are quite loose, but did you have a look at how other parsers might be mocked (if we even do that?). 

If yes, then it would be nice to stay consistent in between the different parsers.

Reply:
To this I've taken some inspiration from the ENTSOE tests and extended them, so they are in line with what we have (though we could improve what we have a bit).

Reply:
alright goodonya